management/clipper_manager.py

```diff
@@ -278,26 +278,28 @@ def register_application(self,
         r = requests.post(url, headers=headers, data=req_json)
         print(r.text)
 
-    def list_apps(self):
-        """List the names of all applications registered with Clipper.
+    def get_all_apps(self, verbose=False):
+        """Gets information about all applications registered with Clipper.
+
+        Parameters
+        ----------
+        verbose : bool
+            If set to False, the returned list contains the apps' names.
+            If set to True, the list contains application info dictionaries.
+            These dictionaries have the same attribute name-value pairs that were
+            provided to `register_application`.
 
         Returns
         -------
-        str
-            The string describing each registered application. If no
-            applications are found, an empty string is returned.
+        list
+            Returns a list of information about all apps registered to Clipper.
+            If no apps are registered with Clipper, an empty list is returned.
         """
-        with hide("output", "running"):
-            result = local(
-                ("redis-cli -h {host} -p 6379 -n {db} keys \"*\"".format(
-                    host=self.host, db=REDIS_APPLICATION_DB_NUM)),
-                capture=True)
-
-            if len(result.stdout) > 0:
-                return result.stdout
-            else:
-                print("Clipper has no applications registered")
-                return ""
+        url = "http://%s:1338/admin/get_all_applications$" % self.host
```

Comment:
We have a few places where we hardcode 1337 or 1338. Can you replace these with `CLIPPER_QUERY_PORT` and `CLIPPER_MANAGEMENT_PORT` respectively?